pom.xml

```diff
@@ -49,4 +49,29 @@
             <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
         </repository>
     </repositories>
+
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <!-- Use the latest released version:
+                    https://repo1.maven.org/maven2/com/github/eirslett/frontend-maven-plugin/ -->
+                    <version>1.6</version>
+                    <configuration>
+                        <nodeVersion>v5.6.0</nodeVersion>
+                        <!-- optional: with node >4.0.0 will use npm provided by node distribution -->
+                        <!-- <npmVersion>2.15.9</npmVersion> -->
+                        <!-- must use http for NCL builds -->
```

Comment:
we should use https, not http (that could be override if you don't need http but default is secure)

Reply:
No, see https://docs.engineering.redhat.com/display/JP/Running+NPM+and+Yarn+builds+with+maven+frontend+plugin